common/core/src/authorization.ts

```diff
@@ -76,6 +76,10 @@ export interface TransportConfig {
    * IP address or internet name of the host machine working as a device or protocol gateway.  Used when communicating with Azure Edge devices.
    */
   gatewayHostName?: string;
+  /**
+   * Public certificate for the CA to use when validating the connection to the server
+  */
+  ca?: string;
```

Comment:
wouldn't this thing belong in the x509 options? or are we keeping some x509 options for iot hub and others for edge?